ios/PurchasesHybridCommon/PurchasesHybridCommonTests/CustomerInfo+TestExtensions.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  CustomerInfo+TestExtensions.swift
+//  PurchasesHybridCommonTests
+//
+//  Created by Will Taylor on 5/27/22.
+//  Copyright © 2022 RevenueCat. All rights reserved.
+//
+
+import Foundation
+import RevenueCat
+
+extension CustomerInfo {
+    static func fromJSON(_ jsonString: String) -> CustomerInfo {
+        let decoder = JSONDecoder()
+        decoder.keyDecodingStrategy = .convertFromSnakeCase
+        decoder.dateDecodingStrategy = .iso8601
+
+        let data = Data(jsonString.utf8)
+        return try! decoder.decode(CustomerInfo.self, from: data)
```

Comment:
:warning: This will make the tests _crash_ instead of _fail_, which means none of the other tests will run and execution stops, instead of reporting each individual failure.
Instead, make this function `throws`, and use `try`. At the call site, you can just do `try CustomerInfo.fromJSON("")` inside of an `it`, and if it throws, the test fails instead.